packages/tx-construction/package.json

```diff
@@ -56,6 +56,7 @@
     "@cardano-sdk/util": "workspace:~",
     "@cardano-sdk/util-rxjs": "workspace:~",
     "@cardano-sdk/wallet": "workspace:~",
+    "delay": "^5.0.0",
```

Comment:
it looks like delay should be part of devDependencies, since it's being used only in the test

Reply:
Nice catch, I intended to add it do devDependencies but failed :grin: 1523fcebe953a529e076434bebf74c020a5485fd